components/http_foundation/introduction.rst

```diff
@@ -140,7 +140,7 @@ has some methods to filter the input values:
 :method:`Symfony\\Component\\HttpFoundation\\ParameterBag::filter`
     Filters the parameter by using the PHP :phpfunction:`filter_var` function.
 
-All getters takes up to three arguments: the first one is the parameter name
+All getters take up to two arguments: the first one is the parameter name
```

Comment:
@xabbuh, I see your point. I reverted the text to its original state plus removal of the unnecessary 's' character.